projects/storefrontlib/shared/components/media/media.component.spec.ts

```diff
@@ -1,13 +1,50 @@
-import { Pipe, PipeTransform } from '@angular/core';
+import {
+  Directive,
+  Input,
+  Pipe,
+  PipeTransform,
+  TemplateRef,
+  ViewContainerRef,
+  inject,
+} from '@angular/core';
 import { TestBed } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
+
+import { InjectionToken } from '@angular/core';
+import { FeatureConfigService } from '@spartacus/core';
 import { MediaComponent } from './media.component';
 import { ImageLoadingStrategy, Media } from './media.model';
 import { MediaService } from './media.service';
 import { USE_LEGACY_MEDIA_COMPONENT } from './media.token';
 
+export const IS_CONFIGURABLE_MEDIA_COMPONENT = new InjectionToken<boolean>(
+  'IS_CONFIGURABLE_MEDIA_COMPONENT'
+);
```

Comment:
Yes, it is used in function configureTestingModule(). The only thing - it shouldn't be exported, since it is used only in tests to configure testing module on the fly.

Reply:
Actually it changed i different scenarios to false `configureTestingModule(new MockMediaService('srcset'), false, true);`

Reply:
Only now I see it's used in `configureTestingModule` function which is used in tests to create specific env. Apologies!